config/schema/rules.schema.yml

```diff
@@ -194,9 +206,21 @@ rules_expression.rules_reaction_rule:
     uuid:
       type: string
       label: 'UUID'
-    event:
-      type: string
-      label: 'Event'
+    events:
```

Comment:
unrelated, this seems wrong  / outdated. Probably we can remove rules_expression.rules_reaction_rule - the plugin does not exist any more afair.